docker-compose.yml

```diff
@@ -22,7 +22,6 @@ services:
     container_name: swe574_term_project  # name the container as swe574_term_project
     command: >
       sh -c "sleep 5 &&
-             python manage.py makemigrations &&
              python manage.py migrate &&
```

Comment:
I deleted this for a while since we are generating extra migrations and I couldn't solve the problem yet.